package/MDAnalysis/analysis/pca.py

```diff
@@ -390,12 +450,13 @@ def rmsip(self, other, n_components=None):
 
         See also
         --------
-        rmsip
+        :func:`rmsip`
```

Comment:
Builds a link to itself (even though labeled func and not meth), needs to be
```suggestion
        :func:`~MDAnalysis.analysis.pca.rmsip`
```